### PR TITLE
Add support for Server-Translations - Add a way to send translated message to player in their client language.

### PR DESCRIPTION
**Pull request by arthurbambou**
This add an easy way to send translated message when Server-Translations is on the server.
If Server-Translations is not on the server, it will send a default message.

There is probably a better way to do it, feel free to suggest any idea ^^.

Tested outside of dev environment.

### CODE REVIEW

File: src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java [hunk 1]
```diff
@@ -210,8 +211,10 @@ private static int startGame(CommandContext<ServerCommandSource> context) throws
                 Text error = startResult.getError();
                 message = error.shallowCopy().formatted(Formatting.RED);
             } else {
-                message = source.getDisplayName().shallowCopy().append(" has started the game!")
-                        .formatted(Formatting.GRAY);
+                message = new TranslatableLiteralText(source.getDisplayName().shallowCopy().append(" has started the game!"),
```
Comment (gegy1000) [inline src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:214]: Hmm- would it be possible to have the messages not duplicated?

Comment (arthurbambou) [reply src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:214]: It's the default message used when Server-Translation is not on the server.

Comment (arthurbambou) [reply src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:214]: It depends if you prefer Server-Translations to be a hard dependency or a soft one.

Comment (gegy1000) [reply src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:214]: Ah okay, I understand. Might it make sense to have the mod be JIJ'd? Or if not JIJ'd completely, a small part of the API that delegates to en_us translation if not installed?

Comment (arthurbambou) [reply src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java:214]: Opened #39 for the not JIJ'd completely case.

**Pull request closed without merging**